### DISCUSSION TITLE
creating feature branches

### DISCUSSION THREAD

**Discussion by Kade03** [Kade03/Tafe22S203#4] [Q&A]
since ive made both of you collaborators does this give you the ability to add your own branches to the github repository? also i think we should just use the normal naming conventions given for each feature branch so theres no confusion e.g feature 1.1 would be mortgage calculator and feature 1.2 would be currency convertor
![Screenshot 2022-09-12 221040](https://user-images.githubusercontent.com/112869678/189656153-8821e3b8-76ef-49a6-afbb-464cfeaef717.png)

**Comment by TafeidMatt** [comment]
We need a release branch.
![image](https://user-images.githubusercontent.com/112943875/190306960-e9bc516d-d575-46c9-94e0-bf5a625b38e3.png)

It is the step after the dev branch.
